spec/support/active_record/schema.rb

```diff
@@ -63,6 +67,8 @@
     t.column :associated_type, :string
     t.column :user_id, :integer
     t.column :user_type, :string
+    t.column :agency_id, :integer
+    t.column :agency_type, :string
```

Comment:
Adding an `agency_type` column seems a little weird, but I think it's correct to not assume the ActiveRecord model type in case we break out more apps in the future that use this column for something different.